### PR TITLE
Pbi about page

### PR DESCRIPTION
**Pull request by swapnilp0x00**
- Added resolver for about page
- Populating page with actual data

### CODE REVIEW

File: website/projects/scimaps/src/app/shared/components/profile-gallery/profile-gallery.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@ <h2 class="title" *ngIf="title !== ''">{{ title }}</h2>
 
 <div class="profile-container">
   <div class="profile" *ngFor="let profile of profiles" (click)="goToLink(profile.link)">
-    <img class="profile-image" [src]="getImageSource(profile)" [alt]="profile.slug + '/image.png'">
+    <img class="profile-image" [src]="getImageSource(profile)" [alt]="profile.slug + '/'+ profile.image">
```
Comment (bherr2) [inline website/projects/scimaps/src/app/shared/components/profile-gallery/profile-gallery.component.html:6]: Move the creation of the 'real image link' to the resolver. Then profile.image will be the actual (relative) url to use.

File: website/projects/scimaps/src/app/shared/components/profile-gallery/profile-gallery.component.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export class ProfileGalleryComponent {
   @Input() description = '';
 
   getImageSource(profile: Profile): string {
-    return `assets/${this.directory}/${profile.slug}/image.png`;
+    return `assets/${this.directory}/${profile.slug}/${profile.image}`;
```
Comment (bherr2) [inline website/projects/scimaps/src/app/shared/components/profile-gallery/profile-gallery.component.ts:20]: Resolver should create the url and thus  profile.image is all that is needed for the image src url.

**Review by bherr2 — CHANGES_REQUESTED** [review]

**Comment by bherr2** [comment]
* [ ] update netlify cms seettings for about (in config.yml line 16) to have data for each person type (curator, etc)
* [ ] create about.md file in correct place
* [ ] Load about.md from resolver
* [ ] use in component

File: website/projects/scimaps/src/app/shared/components/profile-gallery/profile-gallery.component.ts (hunk 2 above)
Comment (swapnilp0x00) [reply website/projects/scimaps/src/app/shared/components/profile-gallery/profile-gallery.component.ts]: Moved this logic to the resolver. Now using a profile.image to directly access the image path.

**Comment by bherr2** [comment]
Great job getting the testing and such together, @swapnilp0x00 . Just address the 3 code smells here: https://github.com/cns-iu/scimaps/pull/71#issuecomment-838961109 and it'll be good to go.

**Review by bherr2 — APPROVED** [review]
Looks good. Mark ready for merging when you want me to merge it.

**Pull request merged**